_posts/2024-03-04-geoserver-2-25-RC-released.md

```diff
@@ -0,0 +1,247 @@
+---
+author: Jody Garnett
+date: 2024-03-04
+layout: post
+title: GeoServer 2.25-RC Release
+categories:
+- Announcements
+tags:
+- Release
+- Release Candidate
+release: release_225
+version: 2.25-RC
+jira_version: 16901
+--- 
+
+GeoServer [2.25-RC](/release/2.25-RC/) release is now available
+with downloads
+([bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25-RC/geoserver-2.25-RC-bin.zip/download),
+[war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25-RC/geoserver-2.25-RC-war.zip/download),
+[windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25-RC/GeoServer-2.25-RC-winsetup.exe/download)), along with 
+[docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25-RC/geoserver-2.25-RC-htmldoc.zip/download) and
+[extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25-RC/extensions/).
+
+This is a release candidate intended for public review and feedback.
+GeoServer 2.25-RC is made in conjunction with GeoTools 31-RC, and GeoWebCache 1.25-RC. 
+
+Thanks to Jody Garnett (GeoCat) for making this release.
+
+## Why share a release candidate?
+
+A sensible question to ask is why a "release candidate" is being produced at all - when we do not recommend running such a thing in production.
+
+GeoServer also follows a "[release early, release often](https://en.wikipedia.org/wiki/Release_early,_release_often)" approach which is where the project shares releases with so  **you can test and provide feedback**.
+
+This results in a lovely balance:
+
+* There are not that many developers of GeoServer, do not have lot of GeoSpatial data to test with.
```

Comment:
```suggestion
* There are not that many developers of GeoServer, do not have lot of geospatial data to test with.
```

However I'm not sure what you have in mind. Are you saying the developers don't have a lot of data?